### PR TITLE
Initial Work.

### PR DESCRIPTION
**Pull request by bscutt**
Initial work from the tentacles editor, tidied with comments and some bugs fixed.

### CODE REVIEW

File: src/scss/emoji-picker.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
+$highlight: #81409b;
+
+.emoji-picker {
```
Comment (jonathanbull) [inline src/scss/emoji-picker.scss]: Should we match the 4 space indentation used in the other files?

File: src/js/emoji-picker.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
+/*!
+ * EmailOctopus EmojiPicker v1.0.0 (https://emailoctopus.com/)
+ * License MIT
```
Comment (bscutt) [inline src/js/emoji-picker.js]: @jonathanbull - we should decide what license we're going to use (if we're going to open source it) - MIT is the most common one.

Comment (jonathanbull) [reply src/js/emoji-picker.js]: MIT is good for me

**Comment by jonathanbull** [comment]
If we are making this a direct replacement for Froala's emoticons library, I think we should also name our `css` and `js` files to match:

screen shot 2018-07-24 at 10 58 29

so that they simply need to switch `plugins` to `third_party` in the file path. Otherwise we can't really advertise it as a drop-in replacement.

But then that _would_ mean we have to name the files `emoticons.*`, which is a bit lame as that's not the name of our project. I'm beginning to think we should just set the ID to `emojis` to keep things simple and differentiate our plugin - the process of adding plugins is quite simple so I don't think people will struggle. It also avoids any expectations that the Froala `emoticon` settings will apply to our plugin (which even if they currently do, may not work in the future).

What do you think? Either way we should add a quick `README.md` (sorry, know it's boring).

**Pull request merged**